packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -272,20 +284,11 @@ async function onMount() {
       {},
     );
   }
-  isLoadingNode.value = false;
-  isLoadingName.value = false;
-  isLoadingRent.value = false;
+  updateLoadingTableValue(true, false);
```

Comment:
IMO. To improve code readability and maintainability, consider introducing an object instead of using named parameters.

```ts
// Define a type for the options
type ContractsLoadingOptions = {
  updateAllTables: boolean;
  loading: boolean;
  contractType?: boolean;
}

// Function using the options object
function updateContractsLoading(options: ContractsLoadingOptions) {}

// Example call
updateContractsLoading({
  updateAllTables: true,
  loading: false,
});
```